client/app/scripts/stores/app-store.js

```diff
@@ -44,6 +44,7 @@ function makeNode(node) {
 
 // Initial values
 
+let changingTopologies = false;
```

Comment:
What a great idea! Sometimes one needs to take a step back to see the simple solution.